src/App.tsx

```diff
@@ -1,27 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [users, setUsers] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const handleAllGoods = async () => {
+    const users1 = await getAll();
```

Comment:
I think it will be great to name const allGoods, redGoods, 5firstGoods instead of users1, users2, users 3 - for better understanding.
Also in state - const [goods, setGoods] = useState<Good[]>([]) - because we are working with goods, not users